### PR TITLE
Variable input/status

### PR DESCRIPTION
**Pull request by tomershk**
### 🔦 Summary

VariableInput - `status` and `statusMessage` props

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @shlomitc @nitayneeman 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

File: src/VariableInput/docs/index.story.js [hunk 1]
```diff
@@ -92,6 +100,21 @@ export default {
...
+          columns([
+            description({
+              title: 'Status',
+              text:
+                'Add an status (error/warning) indication with possible tooltip',
+            }),
+
+            code({
+              compact: true,
+              source: `<>${statusExample('error', 'Please fix this')}
```
[the remark this answers is not in the document]
Comment (tomershk) [reply src/VariableInput/docs/index.story.js]: Will do, trying to following existing code conventions 
https://github.com/wix/wix-style-react/blob/VariableInput/rows/src/Tag/docs/examples.js#L10
😄

File: src/VariableInput/test/VariableInput.spec.js [hunk 2]
```diff
@@ -146,7 +147,40 @@ describe('VariableInput', () => {
...
   });
+  describe('Error', () => {
+    it('should render the error indicator', async () => {
+      const driver = createDriver(<VariableInput status="error" />);
+      expect(await driver.hasError()).toBe(true);
+    });
+
+    it('should render a tooltip with the error message', async () => {
+      const errorMessage = 'Some error';
+      const render = createRendererWithUniDriver(publicDriverFactory);
```
[the remark this answers is not in the document]
Comment (tomershk) [reply src/VariableInput/test/VariableInput.spec.js]: Tooltip driver failed while using the createDriver, took this example from https://github.com/wix/wix-style-react/blob/VariableInput/rows/src/RichTextInputArea/test/RichTextInputArea.spec.js#L119

**Comment by nitayneeman** [comment]
We need to solve the conflicts

**Comment by tomershk** [comment]
Conflicts resolved

File: src/VariableInput/VariableInput.st.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 :import {
     -st-from: "../Foundation/stylable/colors.st.css";
-    -st-named: D80, B30, B10, F00, B50, D40, D10-30, D60
+    -st-named: D80, B30, B10, F00, B50, R10, D40, R30, D10-30, D60
 }
 :import {
   -st-from: '../Foundation/stylable/typography.st.css';
```
Comment (EgoziE) [inline src/VariableInput/VariableInput.st.css]: Why don't you use <Text/> component instead?
I'm in the middle of a typography refactor, and I got some conflicts with this component..

Comment (EgoziE) [reply src/VariableInput/VariableInput.st.css]: This is my PR - https://github.com/wix/wix-style-react/pull/4840
I have small conflicts with `<VariableInput/>`

**Pull request merged**